wikiclass/feature_lists/wikidatawiki.py

```diff
@@ -21,6 +32,183 @@ class items:
     """
     HUMAN = 'Q5'
 
+class Revision:
+    
+    global name
```

Comment:
@halfak : it's for obtaining the variable name from the global variable. In other words, 'global name' contains the similar value that is contained in the variable "name". The value is "wikidatawiki".